src/Components/LandingPage/LandingPage.tsx

```diff
@@ -102,6 +108,10 @@ const LandingPage = ({ setFetchedScreen, handleCheckboxChange }: LandingPageProp
     );
   };
 
+  const createAgeCheckboxLabel = () => {
+    return <FormattedMessage id="disclaimer-label-age" defaultMessage="I confirm I am 13 years of age or older." />;
+  };
```

Comment:
I don't think it is necessary to have a function for this. I think we could just put the FormattedMessage where this function is called.